### PR TITLE
[8.0] porting l10n it vat registries

### PR DESCRIPTION
**Pull request by eLBati**

### CODE REVIEW

**Comment by stevech091** [comment]
I have tested it and it works.

**Comment by eLBati** [comment]
@LeartS is this ok for you? Can we merge?

File: l10n_it_vat_registries/vat_registry.py [hunk 1]
```diff
@@ -152,19 +180,14 @@ def __init__(self, cr, uid, name, context):
...
-                       parser=Parser)
-report_sxw.report_sxw('report.registro_iva_corrispettivi',
-                       'registro_iva_corrispettivi',
-                       'addons/l10n_it_vat_registries/templates/registro_iva_corrispettivi.mako',
-                       parser=Parser)
+        return super(Parser, self).set_context(
+            objects, data, ids, report_type=report_type)
+
+
+class report_registro_iva_venidte(osv.AbstractModel):
```
Comment (tafaRU) [inline l10n_it_vat_registries/vat_registry.py]: Ok it's just abstract class to embed old style report but ```report_registro_iva_vendite``` is preferable. Please fix the typo, thanks.

Comment (eLBati) [inline l10n_it_vat_registries/vat_registry.py]: >
> Ok it's just abstract class to embed old style report but
> report_registro_iva_vendite is preferable. Please fix the typo, thanks.
>
>
>
Done Thanks

**Comment by tafaRU** [comment]
@eLBati could you please rebase to include https://github.com/OCA/l10n-italy/commit/92382e74eda57b81ae9ffc0f55d30cbe2f1e466f?

**Comment by eLBati** [comment]
On 21 May 2015 at 12:25, Alex Comba <notifications@github.com> wrote:

> @eLBati <https://github.com/eLBati> could you please rebase to include
> 92382e7
> <https://github.com/OCA/l10n-italy/commit/92382e74eda57b81ae9ffc0f55d30cbe2f1e466f>
> ?
>
>
>
Merged 8.0 into this (because rebase gave conflicts)

**Comment by tafaRU** [comment]
@hurrinico please see https://github.com/hurrinico/l10n-italy/pull/11

**Comment by sergiocorato** [comment]
@eLBati it works for me
To solve overlapping issue for more than one page of invoices (https://github.com/wkhtmltopdf/wkhtmltopdf/issues/1524) I'd put into l10n_it_vat_registries.css the row below:
tr {
    page-break-inside: avoid;
}

**Comment by eLBati** [comment]
Thanks @sergiocorato , added to qweb directly https://github.com/eLBati/l10n-italy/commit/6a5e42f4ab785a1c9e7fefe3b9fcdd579fbe6f9a to not modify every ```tr``` behavior

**Comment by DAMilani** [comment]
Questa PR è stata sostituita dalla seguente:

https://github.com/OCA/l10n-italy/pull/124

**Comment by Giuliano69** [comment]
Hi Elbati,
is l10n it vat registries now merged in  Italian localization, or still need to be installed separately ?

**Comment by tafaRU** [comment]
@Giuliano69 this PR has been replaced by https://github.com/OCA/l10n-italy/pull/124 so it needs to be installed separately.

**Pull request closed without merging**